### DISCUSSION TITLE
Tool to create pivot tables

### DISCUSSION THREAD

**Discussion by kurtlawrence** [precision-mining-consulting/merz#21] [Ideas]
Potentially create pivot tables via XML construction (ughhh).

See below for scaffold:

```xml
<Report Name="New Pivot Table 2" Layout="&lt;PivotGridLayout&gt;  &lt;Field Name=&quot;Field Custom Field foo\Coal Tonnes&quot;  Visible=&quot;true&quot; Area=&quot;DataArea&quot; AreaIndex=&quot;0&quot; /&gt; &lt;Field Name=&quot;Field Operating Hours&quot;  Visible=&quot;true&quot; Area=&quot;DataArea&quot; AreaIndex=&quot;1&quot; /&gt;  &lt;/PivotGridLayout&gt;" />
```

**Comment by kurtlawrence** [comment]
The more I think about this tool, the more useful I see it becoming.

I think aim for a textual based input, with a goal that we could read a config file off disk and generate a whole bunch of pivot tables (as files) for bulk import.

**Comment by npdelaney** [comment]
https://github.com/precision-mining-consulting/pm-spry-tools/pull/207
